subprojects/android-test/ui-testing-core-app/build.gradle.kts

```diff
@@ -93,76 +91,82 @@ instrumentation {
         fromSource.excludeFlaky = true
     }
 
-    val credentials = project.kubernetesCredentials
-    if (credentials !is Service && credentials !is KubernetesCredentials.Config) {
-        // todo fix this in MBS-11834
-        logger.warn("Instrumentation tasks are not created because kubernetes credentials not set")
-    } else {
-
-        afterEvaluate {
-            tasks.named("check").dependsOn(tasks.named("instrumentationUi"))
+    environments {
+        register<KubernetesViaCredentials>("kubernetes") {
+            token.set(getOptionalStringProperty("kubernetesToken"))
+            caCertData.set(getOptionalStringProperty("kubernetesCaCertData"))
+            url.set(getOptionalStringProperty("kubernetesUrl"))
+            namespace.set(getOptionalStringProperty("kubernetesNamespace"))
         }
+    }
 
-        val emulator22 = CloudEmulator(
-            name = "api22",
-            api = 22,
-            model = "Android_SDK_built_for_x86",
-            image = emulatorImage(22, "7bb4b0b720"),
-            cpuCoresRequest = "1",
-            cpuCoresLimit = "1.3",
-            memoryLimit = "4Gi"
-        )
+    experimental {
+        fetchLogcatForIncompleteTests.set(true)
```

Comment:
So, we should discuss this experiment. Could we delete it and make it as a default behaviour

Reply:
So we forgot to discuss that
@RuslanMingaliev Could you initiate the discussion?